src/bundle/Resources/public/js/OnlineEditor/toolbars/config/base-fixed.js

```diff
@@ -10,8 +10,9 @@ export default class EzConfigFixedBase extends EzConfigBase {
         const editorRect = editor.element.getClientRect();
         const toolbarHeight = toolbar ? toolbar.getBoundingClientRect().height : 0;
         const shouldBeFixed = editorRect.top - toolbarHeight - 2 * TOOLBAR_OFFSET < 0;
+        const header = document.querySelector('.ez-edit-header__content-type-name');
```

Comment:
I think we don't have this header in master - 3.2 so I think this is not needed.

Reply:
Done in https://github.com/ezsystems/ezplatform-richtext/pull/148/commits/914751514f163c8b0cb63f92828bd9e8e955cf9e